tenable/io/v3/vm/networks/api.py

```diff
@@ -0,0 +1,313 @@
+'''
+Networks
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`networks <networks>` API endpoints.
+
+Methods available on ``tio.v3.vm.networks``:
+
+.. rst-class:: hide-signature
+.. autoclass:: NetworksAPI
+    :members:
+'''
+from typing import Dict, List, Union
+from uuid import UUID
+
+from restfly.utils import dict_clean
+
+from tenable.errors import UnexpectedValueError
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.vm.networks.schema import NetworkSchema
+
+
+class NetworksAPI(ExploreBaseEndpoint):
+    '''
+    This class contains methods related to Networks API
+    '''
+    _path = 'api/v3/networks'
+    _conv_json = True
+    _schema = NetworkSchema()
+
+    def create(self,
+               name: str,
+               description: str = '',
```

Comment:
I believe this should default to `None`, as it is not a required field

Reply:
In v2 endpoint of pyTenable It was originally None but before passing the value to payload it was updated to '',
```
if not description:
       description = ''
```

ref https://github.com/dhvanilshah-tenable/pyTenable/blob/develop/tenable/io/networks.py#L64-L65

hence we decided to keep the default value as `''` in here and in schema otherwise we would need to handle it in post_dump Schema.

Also both (None and '') are supported by the API, the difference is if we pass `''` as description there will be a key-value `'description':''` in the response dict.

Reply:
Ok, I suppose that's ok then. I would imagine the creation would work the same for the API regardless if it is an empty string or `None`. Let's verify that, and if that's the case, I'm fine with your approach